styleguide/derek/global-components/tracking/tracking.js

```diff
@@ -197,14 +197,18 @@ Zoolander.Tracking = (function Tracking($) {
     });
 
     $(document).on('click', '.track-signupMenu', (e) => {
-      const protocol = $(e.target).prop('protocol');
-      const hostname = $(e.target).prop('hostname');
-      const pathname = $(e.target).prop('pathname');
+      const protocol = $(e.currentTarget).prop('protocol');
+      const hostname = $(e.currentTarget).prop('hostname');
+      const pathname = $(e.currentTarget).prop('pathname');
+      const hash = $(e.currentTarget).prop('hash');
+      const port = $(e.currentTarget).prop('port');
+      const portStr = port ? `:${port}` : '';
```

Comment:
Note: I did a little bit more than just switching to e.currentTarget here. Specifically, I added the ability to add the port and the URL hash to the tracked URL when applicable. The reason for that, is so that I could just attribute some links to http://localhost:9876 instead of actually clicking through to actual URLs.